elastic/integrations
# Threat Intelligence Dashboard fix

## What does this PR do?

Updating the dashboard to point to data views by name instead of a GUID.

## How to test this PR locally

Install the assets from the integration and verify that the dashboard doesn't show an error like `Could not find the data view: b2a3b629-f629-4f2d-b437-9640f850312d`

## Related issues

Relates https://github.com/elastic/security-team/issues/4706

## Screenshots

Was: 
<img width="1770" alt="Screen Shot 2022-08-31 at 6 27 32 PM" src="https://user-images.githubusercontent.com/9203126/187796119-683d4d49-6300-4f56-92b0-b602376ffaa4.png">

Now is:
<img width="1765" alt="Screen Shot 2022-08-31 at 6 27 39 PM" src="https://user-images.githubusercontent.com/9203126/187796136-be920e4b-e39f-47e3-9c2c-90a3379a1cba.png">
